autoload/deoplete/init.vim

```diff
@@ -105,6 +105,10 @@ function! deoplete#init#_variables() abort "{{{
   call deoplete#util#set_default(
         \ 'g:deoplete#enable_debug', 0)
   call deoplete#util#set_default(
+        \ 'g:deoplete#debug_log', '/tmp/deoplete.log')
```

Comment:
`/tmp/deoplete.log` is not Windows compatible.
It's vim's `tempname()` and `deoeplete.log` file name better than it.